app/amity.py

```diff
@@ -0,0 +1,328 @@
+from database import Office as Ofisi, LivingSpace as Hostel, Person as Mtu
+from person import Fellow, Staff
+from rooms import LivingSpace, Office
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import random
+from database import Base
+
+
+class Amity(object):
+    '''This class contains all rooms and all people in amity'''
+
+    def __init__(self):
+        self.all_rooms = []
+        self.office_rooms = []
+        self.livingspaces = []
+        self.all_people = []
+        self.fellows = []
+        self.staff = []
+        self.allocated = []
+        self.unallocated = []
+
+    def create_room(self, name, type_room):
+        '''this method create rooms.it can create multiple rooms'''
+        if type_room.lower() == 'livingspace':
+            room = LivingSpace(name)
+            self.livingspaces.append(room)
```

Comment:
the `Room` class can have `room_type` attribute so that you can just have the `all_rooms` list.